lib/RealWlanInterface.hpp

```diff
@@ -49,19 +49,19 @@ class RealWlanInterface: public IWlanInterface
     const std::shared_ptr<Wlansvc::WlanApiWrapper> m_wlansvc;
     const GUID m_interfaceGuid;
 
-    std::mutex m_promiseMutex;
+    mutable std::mutex m_promiseMutex;
```

Comment:
Would be very helpful if there was documentation describing what this (and `m_notifMutex`) protect.

Reply:
Thanks!